src/components/Person/Person.jsx

```diff
@@ -1 +1,32 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  let personAge = null;
+  let partner = 'I am not married';
+
+  if (age) {
+    personAge = `I am ${age}`;
+  }
+
+  if (isMarried) {
+    if (sex === 'm') {
+      partner = `${partnerName} is my wife`;
+    } else if (sex === 'f') {
+      partner = `${partnerName} is my husband`;
+    }
+  }
```

Comment:
```suggestion
  const personPartner = sex === 'm' ?  'wife' : 'husband';
  cosnt partner = isMarried ? ${partnerName} is my ${personPartner}` : 'I am not married';
```
it will be shorter and more readeable.